nbviewer/providers/local/handlers.py

```diff
@@ -168,7 +153,17 @@ def get(self, path):
         if is_download:
             self.download(fullpath)
             return
+        
+        return fullpath
 
+    @gen.coroutine
+    def deliver_notebook(self, fullpath, path):
+        """
+        breadcrumbs: list of dict
+            Breadcrumb 'name' and 'url' to render as links at the top of the notebook page
+        title: str
+            Title to use as the HTML page title (i.e., text on the browser tab)
```

Comment:
`breadcrumbs` and `title` are not parameters in the `deliver_notebook` function. Let's at least remove them from the docstring and consider documenting the actual parameters.